recipes/ruby.rb

```diff
@@ -32,11 +32,13 @@
   node.set['build-essential']['compile_time'] = true
   include_recipe "build-essential"
 
-  if node['postgresql']['enable_pgdg_yum']
+  if node['postgresql']['enable_pgdg_yum'] || node['postgresql']['use_pgdg_packages']
     package "ca-certificates" do
```

Comment:
We still need the ca-certificates package installed if we're using pgdg originated packages without wanting to set the ```node['postgresql']['enable_pgdg_yum']``` attribute to true